js/apps/admin-ui/pom.xml

```diff
@@ -136,7 +136,9 @@
       "resourceUrl": "${resourceUrl}",
       "masterRealm": "${masterRealm}",
       "resourceVersion": "${resourceVersion}",
-      "isRunningAsTheme": true
+      "isRunningAsTheme": true,
+      "logo": "${properties.logo!""}",
+      "logoUrl": "${properties.logoUrl!""}"
```

Comment:
Since this URL is used as a hypertext reference perhaps it would be more accurately named `logoHref`? This matches the naming convention of the [`href`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href) attribute and is well known amongst web devs.

Reply:
Again, good point, but it's the same property from account console.

Reply:
See https://github.com/keycloak/keycloak/pull/20201#discussion_r1193669373